ci/taos/plugins-base/pr-audit-build-android.sh

```diff
@@ -82,22 +96,27 @@ function pr-audit-build-android-run-queue(){
     echo "Running 'source /etc/environment'"
     source /etc/environment
 
+    # Set the path about Android build tools
+    export ROOT_ANDROID_CI=/var/www/html/android
+
     # Android NDK
-    export ANDROID_NDK=/var/www/html/android/android-ndk-r12b
+    export ANDROID_NDK=$ROOT_ANDROID_CI/android-ndk-r12b
     export PATH=$ANDROID_NDK:$PATH
     echo "Exporting an ANDROID_NDK path ..."
     echo $PATH
     ndk-build --help
 
     # gst-android prebuilt binary (e.g., .a, .so, .h)
-    export GSTREAMER_ROOT_ANDROID=/var/www/html/android/gst_root_android/
+    export GSTREAMER_ROOT_ANDROID=$ROOT_ANDROID_CI/gst_root_android/
     echo "Exporting a GSTREMER_ROOT_ANDROID path..."
     echo $GSTREAMER_ROOT_ANDROID
 
     # Check if dependent packages are installed. Please add required packages here.
     check_dependency sudo
     check_dependency curl
     check_dependency ndk-build
+    check_dependency sed
+    check_dependency gradle
```

Comment:
Is "**gradle**" command correct? 
As we know, we use "**graldew**" command to build ***.apk** on the terminal.

Reply:
gradlew is a shell script, not a build command.

Reply:
Sure, **gradlew** is a shell script. and **gradle** is a command that is written by JAVA. So, I wonder about the "check_dependency gradle" statement if this statement is appropriate or not

Reply:
I understand. now we are using the studio and gradle is unnecessary.
Now I updated the PR to remove dependency.